### DISCUSSION TITLE
How do I add support for additional Distros?

### DISCUSSION THREAD

**Discussion by JackisGT** [PierreGode/Linux-Active-Directory-join-script#80] [Q&A] [answered]
How do I add support for additional Distros? can i do it my self?

**Comment by PierreGode** [comment, marked as answer]
> How do I add support for additional Distros? can i do it my self?

Hi! yes just clone the repository, make your changes and do a pull request and i will review it.

**Comment by xtopherbv** [comment]
Hello.
Supported LMDE 6?

**Reply by PierreGode** [reply]
Hi I will take a look at it.
